pages/index.tsx

```diff
@@ -0,0 +1,560 @@
+import { FunctionComponent } from 'react';
+
+import { H1, H3 } from '../styles/MDComponents';
+import PageLink, { CallToAction } from '../components/PageLink';
+
+import AnalyticsManagerIcon from '../images/svg/analyticsmanager.svg';
+import DevIcon from '../images/svg/dev.svg';
+
+import styles from './index.module.scss';
+
+function emptyRowElement() {
+  return <div style={{ display: 'flex', padding: '15px' }} />;
+}
+
+const LandingPage: FunctionComponent = () => {
+  return (
+    <div style={{ width: '85%' }}>
+      <H1>Getting started with Avo</H1>
+      <p className={styles.text}>
+        We distinguish 3 steps to adopt Avo. You don't need to go through all
+        the steps to receive value, because every step has value in itself.
```

Comment:
The Avo workflows are layered, such that they can be adopted as a whole or in part – depending on your use case. Each layer of the workflow provides value and each of them requires different amount of engineering contibution.